src/utilities.cpp

```diff
@@ -85,8 +85,9 @@ std::vector<int> aste::setupMesh(precice::SolverInterface &interface, const aste
   auto tconnectivity = std::chrono::steady_clock::now();
   VLOG(1) << "Mesh Setup: 2) Edges";
   const auto edgeMap = setupEdgeIDs(interface, mesh, meshID, vertexIDs);
-
-  VLOG(1) << "Mesh Setup: 3) Triangles";
+  if (!mesh.triangles.empty()) {
```

Comment:
We need the same mechanism for the Edges above

Reply:
Not exact same is applicable but a better logic is constructed now